libs/components/storybook/src/lib/preview-wrapper/preview-wrapper.component.ts

```diff
@@ -45,40 +45,46 @@ export class PreviewWrapperComponent implements OnInit, OnDestroy {
     }
   }
   public get theme() {
-    return this._theme;
+    return this.#_theme;
   }
 
   public get themeSettings(): SkyThemeSettings {
-    return this._themeSettings;
+    return this.#_themeSettings;
   }
   public set themeSettings(value: SkyThemeSettings) {
-    this._themeSettings = value;
-    if (this.initialized) {
-      this.themeService.setTheme(this._themeSettings);
+    this.#_themeSettings = value;
+    if (this.#initialized) {
+      this.#themeService.setTheme(this.#_themeSettings);
     }
   }
 
-  private _themeSettings = new SkyThemeSettings(
+  #_themeSettings = new SkyThemeSettings(
     SkyTheme.presets.default,
     SkyThemeMode.presets.light
   );
-  private _theme?: 'default' | 'modern-light' | 'modern-dark';
-  private readonly _ngUnsubscribe = new Subscription();
-  private initialized = false;
+  #_theme: 'default' | 'modern-light' | 'modern-dark' = 'default';
+  #ngUnsubscribe = new Subscription();
+  #initialized = false;
+
+  #themeService: SkyThemeService;
+  #renderer: Renderer2;
 
   constructor(
-    private themeService: SkyThemeService,
+    themeService: SkyThemeService,
     @Inject('BODY') private body: HTMLElement,
```

Comment:
Still need the `private`? If so, maybe this should be an injection token instead.

Reply:
That was just a miss. Fixed.